### DISCUSSION TITLE
robots.txt in github.io pages

### DISCUSSION THREAD

**Discussion by williamadba** [community/community#64865] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Do crawlers honor a robots.txt page in the root of a GitHub.io folder? Any documentation or guidance on this?

**Comment by arafajs** [comment, marked as answer]
Yes, crawlers will honor a robots.txt page in the root of a GitHub.io folder.
GitHub does not provide any official documentation specific to using robots.txt on GitHub Pages. But the standard robots.txt syntax and conventions still apply.

**Reply by arafajs** [reply]
Documentation:
https://developers.google.com/search/docs/crawling-indexing/robots/robots_txt

**Reply by williamadba** [reply]
Thanks. Was concerned about github.io pages and robots.txt specifically, this helps.
